### PR TITLE
Updated my posts tab

### PR DESCRIPTION
**Pull request by Astijq**

### CODE REVIEW

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 1]
```diff
@@ -1,8 +1,49 @@
-﻿<h3>MyPosts</h3>
+﻿@page "/myposts"
+<h3>MyPosts</h3>
 
+<Virtualize Context="post" Items="@posts">
+<table class="table">
+		    <thead>
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:6]: why is this in the middle of the line?

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 2]
```diff
@@ -1,8 +1,49 @@
...
+            <th>Description</th>
+            <th>Measurements</th>
+            <th>Estimated price</th>
+            <th>Adresas</th>
+            <th>Žiūrėti skelbimą</th>
+        </tr>
+    </thead>
+	<tbody>
+		<tr>
+			<td>Date</td>
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:19]: not aligned

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 3]
```diff
@@ -1,8 +1,49 @@
...
+			<td><img src="@post.Image" style="height:200px; max-width: 300px;"/></td>
+			<td>@post.Description</td>
+			<pre>
+Weight: @post.Weight
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:23]: not aligned

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 4]
```diff
@@ -1,8 +1,49 @@
...
+		</tr>
+	</tbody>
+</table>
+	<hr />
+</Virtualize>
 
 
-@code { 
+@code {
+
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:39]: why the empty line???

**Review by neringamaj — APPROVED** [review]

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 5]
```diff
@@ -1,8 +1,49 @@
...
+<Virtualize Context="post" Items="@posts">
+<table class="table">
+		    <thead>
+        <tr>
+            <th>Date uploaded</th>
+            <th>Image</th>
+            <th>Description</th>
+            <th>Measurements</th>
+            <th>Estimated price</th>
+            <th>Adresas</th>
```
Comment (kambalas) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:13]: Kodėl pusę lietuviškai o kiti angliškai?

**Review by kambalas — APPROVED** [review]

**Pull request merged**